### ISSUE TITLE
Responsiveness

### ISSUE THREAD

**Issue by zschrunk** [Firefly9093/web-iv-portfolio-site#6]
Your site is responsive and but it stills has a few bugs that needed to be worked out. When you get to mobile it messes with the alignment issues in the footer. In the footer it centers some stuff but not other. 

Satisfactory

**Comment by e-m-durham** [comment]
I agree with @zschrunk that there are quite a few alignment issues in small screens. 

I would also like to see your navigation minimize in smaller screens like you have in your wireframes. It changes the hierarchy quite a bit and can take up about a third of the screen otherwise. In fact, it might be a good idea to really work out the header in the prototype before moving on to styling.

**Comment by e-m-durham** [comment]
I'd consider making your contact form larger in medium and small screen sizes. It gets pretty small and you can use these multiple breakpoints to your advantage for a tailored viewing experience per device.
